CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0] - 2025-08-26
+
+### Added
+
+- [Common] Added abstract server and registerTools/Resources [#70](https://github.com/SmartBear/smartbear-mcp/pull/70)
+- [Pactflow] Added MCP tool to generate Pact tests [#71](https://github.com/SmartBear/smartbear-mcp/pull/71)
+- [Pactflow] Added MCP tool to fetch provider states for pactflow and pactbroker [#87](https://github.com/SmartBear/smartbear-mcp/pull/87)
+- [Pactflow] Added MCP too to review Pact test [#89](https://github.com/SmartBear/smartbear-mcp/pull/89)
+
+### Changed
+
+- [Pactflow] Added ZodSchema support for parameter passing in generate pacts tool [#90](https://github.com/SmartBear/smartbear-mcp/pull/90)
```

Comment:
Suggest we remove this. As this is the first release of any PactFlow tool, this isn't a relevant changelog entry for users.